packages/lib/storybook/stories/components/UPI.stories.tsx

```diff
@@ -18,9 +18,22 @@ export const UPI: UpiStory = {
     },
     args: {
         countryCode: 'IN',
+        // @ts-ignore Seems like enum isnt the best way to export fixed strings
         componentConfiguration: {
-            // @ts-ignore Seems like enum isnt the best way to export fixed strings
-            defaultMode: 'vpa'
+            apps: [
+                {
+                    id: 'bhim',
+                    name: 'BHIM'
+                },
+                {
+                    id: 'gpay',
+                    name: 'Google Pay'
+                },
+                {
+                    id: 'phonepe',
+                    name: 'Phone Pe'
+                }
+            ]
```

Comment:
Can this be removed, since the Backend now returns the 'apps'?

Reply:
Will do.